### PR TITLE
[UI] `clr-` prefix for css classes

### PR DESCRIPTION
**Pull request by jeeyun**
Using postcss to generate two versions of css files:
- one where every classname is prefixed with `clr-`
- one where old classname and prefixed version coexist to allow the client app to migrate at their own pace

Note that the script order in `build:ui` matters, since one of them change the css files in place as opposed to generating a new file. 

Signed-off-by: Jeeyun Lim <jeeyun.lim@gmail.com>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -13848,6 +13844,69 @@
         }
       }
     },
+    "postcss-prefix-selector": {
```
Comment (gnomeontherun) [inline package-lock.json]: The package-lock changes are more extensive than just this new package. Can you revert the package-lock.json changes, except for this new piece?

Comment (jeeyun) [reply package-lock.json]: I'll take a look and revert.

**Comment by gnomeontherun** [comment]
Just to follow up, we discussed keeping this open until Matt has the other PRs ready so we can see them all together first, but otherwise this is good to me.

**Comment by mathisscott** [comment]
this solution is problematic. we are pushing past 1.0 to find a better way to do this.

**Pull request closed without merging**